test/e2e/Tests/Tests/ExternalEventTests.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Net;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E;
+
+[Collection(Constants.FunctionAppCollectionName)]
+public class ExternalEventTests
+{
+    private readonly FunctionAppFixture _fixture;
+    private readonly ITestOutputHelper _output;
```

Comment:
nit: Unlike the Functions team code, we don't really use `_`-prefixed field names in our code bases. Rather, we use normal caselCasing and use `this.` for referencing.